src/client/integrations.js

```diff
@@ -0,0 +1,97 @@
+var _ = require('slapdash')
+var http = require('@qubit/http-api')
+
+var QFN_DEFAULT_REQUEST_TIMEOUT = require('../../constants').QFN_DEFAULT_REQUEST_TIMEOUT
+var TIMEZONE_OFFSET = new Date().getTimezoneOffset()
+
+// Lifted from https://github.com/qubitdigital/experience-engine/blob/master/lib/util/integration/index.js
+module.exports = function createIntegrationApi (visitorId, experienceMeta, qfnMap = {}, logger) {
+  var meta = {
+    source: 'experience',
+    visitorId: visitorId,
+    experience: {
+      experienceId: experienceMeta.experimentId,
+      iterationId: experienceMeta.iterationId,
+      variationId: experienceMeta.variationId,
+      variationMasterId: experienceMeta.variationMasterId,
+      isPreview: true
+    }
+  }
+  var gatewayUrl = `https://integrations.qubit.com`
```

Comment:
@alexolivier should we use the Qubit API endpoint here?

Reply:
not currently